packages/react-storage/src/components/StorageBrowser/Views/Controls/__tests__/Tablev2.spec.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { TableV2, TableData, ColumnData, RowData } from '../TableV2';
+
+describe('TableV2', () => {
+  const renderColumnItem = (column: ColumnData) => (
+    <th
+      key={column.key}
+      className={column.className}
+      aria-label={column['aria-label']}
+    >
+      {column.children}
+    </th>
+  );
+
+  const renderRowItem = (row: RowData) => (
+    <td key={row.key} className={row.className}>
+      {row.children}
+    </td>
+  );
+
+  it('renders the table with column headers and rows', () => {
+    const data: TableData<RowData, ColumnData> = {
+      columns: [
+        { key: 'column1', children: 'Header 1', className: 'header-class' },
+        { key: 'column2', children: 'Header 2', className: 'header-class' },
+      ],
+      rows: [
+        [
+          {
+            key: 'row1-column1',
+            children: 'Row 1 Column 1',
+            className: 'my-class',
+          },
+          {
+            key: 'row1-column2',
+            children: 'Row 1 Column 2',
+            className: 'my-class',
+          },
+        ],
+        [
+          {
+            key: 'row2-column1',
+            children: 'Row 2 Column 1',
+            className: 'my-class',
+          },
+          {
+            key: 'row2-column2',
+            children: 'Row 2 Column 2',
+            className: 'my-class',
+          },
+        ],
+      ],
+    };
+
+    render(
```

Comment:
Rather than use `screen` for referencing `getByText`, prefer destructuring from the output of `render`:
```
const { getByText } = render(<Hi />);
```

Reply:
Should I also destructure the other methods I used too, like `queryAllByRole` for instance?